client/src/views/files/FoldersPage.vue

```diff
@@ -343,18 +340,19 @@ async function importFiles(): Promise<void> {
 }
 
 function selectAllFiles(checked: boolean): void {
-  for (const item of fileItemRefs.value || []) {
+  for (const item of fileListItemRefs.value || []) {
```

Comment:
Should check what display mode we're in and set the values for either `fileListItemRefs` or `fileGridItemRefs`. I know, you can't select all files in GridMode, but this function might be used to reset the selection at one time and it should handle GridMode too.

Reply:
Ok so I added a `if` condition to check that